miro_preprocessor/generate_tfvars.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Name of tfvars file
+TF_VARS=terraform.tfvars
+RELEASE_IDS_FILE="release_ids.tfvars"
+
+# Ensure we don't have stale variables from a previous run
+rm -f $TF_VARS
+rm -rf releases
+
+# Download releases from S3
+mkdir -p releases
+aws s3 cp s3://platform-infra/releases releases --recursive
+
+# Build a tfvars file containing the release ids
+echo "release_ids = {" >> "$TF_VARS"
+for f in releases/*;
+do
+  echo "Processing $f: $(cat $f)"
+  echo "  $(basename $f) = \"$(cat $f)\"" >> "$TF_VARS"
+done
+echo "}" >> "$TF_VARS"
```

Comment:
Ick. This script now appears three times in the repo – in `shared_infra`, and in `lambdas`. We should put a single, canonical copy in the `terraform` directory.

Reply:
Each project _might_ do this differently (and we do actually). We could extract some of the functionality elsewhere but the scripts are acting as "hooks" to allow each service to implement this however necessary.